### DISCUSSION TITLE
Set Image Size

### DISCUSSION THREAD

**Discussion by hard-game** [cncf/landscape2#756] [Q&A] [answered]
can i set difference image size like landscape?

**Comment by hard-game** [comment]
![Image](https://github.com/user-attachments/assets/7829739a-6946-4a66-b2b6-4c204eb912c4)

**Comment by tegioz** [comment, marked as answer]
Hi @hard-game 👋

In that case, we are using the [featured items setting](https://github.com/cncf/landscape2/blob/58b7b22d5d36e1091d3e99e68b11467feaaa9197/docs/config/settings.yml#L121-L155) to make some of them stand out.

You can also [adjust the size of all items when using the grid view mode from the settings](https://github.com/cncf/landscape2/blob/58b7b22d5d36e1091d3e99e68b11467feaaa9197/docs/config/settings.yml#L194-L204) if you wish.
